### ISSUE TITLE
Add _ID automatic to models 

### ISSUE THREAD

**Issue by jaberbu** [jaberbu/onfirework#16]
Would be great not make mandatory to add `_id` field to Interfaces 

`interface Schema {
    _id?: string | undefined;
    FIELD_1: string;
    FIELD_2: number;
}`

**Comment by jaberbu** [comment]
Options:
![imagen](https://user-images.githubusercontent.com/64438389/99826676-fad03e80-2b58-11eb-9bdc-eee1b8ed12ee.png)
Results:
![imagen](https://user-images.githubusercontent.com/64438389/99826706-04f23d00-2b59-11eb-8118-fb812c8cb5b7.png)

But if you want access to first element you need to know its id:
`results['1ARo7ZoCFcwETOtVjjAI'].BRAND`

**Comment by ezequieltejada** [comment]
I'll take it

**Comment by jaberbu** [comment]
@ezequieltejada 

The idea is add _`id` automatically to the interface created by the user.

Interface created buy user and passed to `Onfirework<Schema>('COLLECTION')`
![imagen](https://user-images.githubusercontent.com/64438389/99915016-648b4c80-2d01-11eb-8121-49c4df6b2395.png)

And gets `_id` too
![imagen](https://user-images.githubusercontent.com/64438389/99915034-7bca3a00-2d01-11eb-8728-cf35a2d32507.png)

**Comment by ezequieltejada** [comment]
Ups! Sorry, I miss understood. Please reject the pull request

**Comment by jaberbu** [comment]
Close #20

**Comment by jaberbu** [comment]
@ezequieltejada bad news :(  works 50%

Works great as Filter suggest 👍 
![imagen](https://user-images.githubusercontent.com/64438389/100136722-b2cc5700-2e8b-11eb-9b1d-11fec6c6cbed.png)

Not work 👎 
![imagen](https://user-images.githubusercontent.com/64438389/100136911-ec04c700-2e8b-11eb-84fa-55e7550f8091.png)

We need reopen an issue to try solved this bug.

**Comment by jaberbu** [comment]
There is also the problem that requires importing Results to declared filters outside the `listDocs(where)`

```
const where:Filter<Result<BikeSchema>>[] = [
    ['BRAND', '==', 'Ducati']
]
```

![imagen](https://user-images.githubusercontent.com/64438389/100137956-96c9b500-2e8d-11eb-959e-fd6af3aa9f16.png)

**Comment by ezequieltejada** [comment]
I'll take a look on it

**Comment by jaberbu** [comment]
@ezequieltejada We should revert type for param Filter to `Filter<T>[]` is not necesary filtering by _ID 

eg: `bikes.listDocs([['_id', '==', '1234']])`

![imagen](https://user-images.githubusercontent.com/64438389/100201849-d54c8780-2f00-11eb-8c1b-18a2a267443d.png)

This is other thing:

```
const where:Filter<Result<BikeSchema>>[] = [['BRAND', '==', 'Ducati']]
const ducati:Promise<Result<BikeSchema>[]> = bike.listDocs(where)
```

**Comment by jaberbu** [comment]
Fixed in Pull Request #22
